src/locale/systemd-localed.rules

```diff
@@ -0,0 +1,12 @@
+// This file is part of systemd.
```

Comment:
For configs shipped in other projects' dir we always add a full license description, including SPDX header and the intro of LGPL, as with `org.freedesktop.network1.policy`.

Could you also fix `systemd-networkd.rules`?